### ISSUE TITLE
Question about using liftoff-generated GFF files directly

### ISSUE THREAD

**Issue by CreekWechsel** [Kuanhao-Chao/LiftOn#30]
Hi,

Your program is fantastic! I would like to know if the GFF files generated by liftoff can be used directly without integrating them with EVM and ab initio annotation results.

Best regards,
Chen

**Comment by Kuanhao-Chao** [comment]
Yes — the LiftOn output GFF3 is a complete, ready-to-use lifted annotation (Liftoff DNA-lift + miniprot protein evidence + ORF-rescue, merged). You can use it directly; feeding it into a downstream integrator (EVM, etc.) is optional and project-specific. Closing.
